sixtyfps_compiler/generator/cpp.rs

```diff
@@ -896,7 +896,7 @@ fn generate_component(
             Access::Private,
             Declaration::Var(Var {
                 ty: "sixtyfps::private_api::WindowRc".into(),
-                name: "window".into(),
+                name: "window_".into(),
```

Comment:
I assume you can't call it `window` because of the new `window()` function.  
But maybe it can have a more descriptive name: `window_rc`, `m_window`, `component_window`, `window_handle`, `window_inner`
What do you think?

Reply:
Good point. I thought of `m_window` but realised we don't use the `m_` prefix, but for some reason (lack of coffee?) I couldn't see the other options. I'll go with `window_rc`.